apps/occurrences/manager.py

```diff
@@ -1,6 +1,7 @@
 from typing import List, Optional
 
 from django.db.models import Prefetch, Q
+from elasticsearch_dsl import Q as Q2
```

Comment:
How about `as ESQuery` for readability?

Reply:
We won't need the other `Q` in there after ES implementation